src/Controller/Backend/General.php

```diff
@@ -197,15 +197,17 @@ public function prefill(Request $request)
 
         if ($request->isMethod('POST') || $request->query->getBoolean('force')) {
             $form->handleRequest($request);
-            if ($form->get('contenttypes')->has('contenttypes')) {
+            if (!empty($form->get('contenttypes')->getData())) {
```

Comment:
The negation will work for an empty array, however after running though xdebug it returns an `[0]` which won't pass negation, but [checking Symfony docs](http://symfony.com/doc/current/reference/forms/types/choice.html#empty-data) when the field type is `ChoiceType` **and** `multiple` is true, **and** `expand is true it should return an empty array … that isn't empty to me … so for now your `empty()` wins as a solution to getting this over the line. Just unsure as to what that line is.